backend/GymBeam/appsettings.json

```diff
@@ -24,6 +24,11 @@
     "https://localhost:5173",
     "http://127.0.0.1:5173",
     "https://127.0.0.1:5173"
-  ]
+  ],
+  "GoogleOAuth": {
+    "ClientId": "1018881299074-p920lgqtm7lm8c6e8g6hn0a0uum7l4d3.apps.googleusercontent.com",
+    "ClientSecret": "ENTER_SECRET_HERE",
```

Comment:
we need to somehow handle this. Maybe keep it in env vars?

Reply:
done